sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -474,4 +474,30 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       await queryRunner.release();
     }
   }
+
+  /**
+   * Search the institution based on the search criteria.
+   * @param legalName legalName of the institution.
+   * @param operatingName operatingName of the institution.
+   * @returns Searched institution details.
+   */
+  async searchInstitution(
+    legalName: string,
+    operatingName: string,
+  ): Promise<Institution[]> {
+    const searchQuery = this.repo
+      .createQueryBuilder("institution")
+      .where("institution.establishedDate is not null");
```

Comment:
This where clause is added to have a default not failing condition and add additional conditions dynamically in the below lines

Reply:
Alternatively, you can use a dummy where clause that will not cause DB processing, like below.
```
.where("1 = 1");
```

Reply:
Yes I thought of it, but I thought, this route can be reused in the future, to fetch all the institution. So thought of a valid scenario. I can change, what you suggest??